docs/development.rst

```diff
@@ -91,8 +91,26 @@ To do both::
 
 .. _compiling-with-gulp:
 
-Compiling on the host system with gulp
---------------------------------------
+Compiling JS on the host system with webpack
+--------------------------------------------
+For a quicker iteration cycle while developing the frontend app you can run::
+
+    npm install
+    npm run webpack:dev
+
+This watches the react frontend and rebuilds both the web and SSR bundle
+when changes occur. It rebuilds only what has changed.
```

Comment:
Perhaps we can also mention here that it takes care of restarting the SSR server?